### PR TITLE
Channel buffer missing Key

### PR DESCRIPTION
**Pull request by klexas**
Key not being passed through while creating channel buffers.

### CODE REVIEW

**Comment by brunnre8** [comment]
This is missing the corresponding changes to the docs.

**Comment by klexas** [comment]
> This is missing the corresponding changes to the docs.

You're right, I missed that, funny enough the docs had the key mentioned for the `join([key])` method already. Updated.

File: docs/clientapi.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ order.
 Request that the IRC server sends a list of available channels. Extra parameters
 will be sent.
 
-##### `.channel(channel_name)`
+##### `.channel(channel_name, key)`
```
Comment (brunnre8) [inline docs/clientapi.md:127]: it's optional, so wrap it in `[]` same as the other commands

Comment (klexas) [reply docs/clientapi.md]: Thanks, need to review the doc specs for future reference.

**Review by MaxLeiter — APPROVED** [review]

**Pull request merged**